python/ray/tests/test_autoscaler_fake_multinode.py

```diff
@@ -100,6 +101,51 @@ def i():
     # __example_end__
 
 
+@pytest.mark.parametrize("autoscaler_v2", [False], ids=["v1"])
```

Comment:
Why do we only test for v1? It's helpful to also test the behavior of v2 if it has already been handled.

Reply:
Unfortunately, v2 hasn't handled it. I am working on it.

Reply:
I may still need some time to figure it out. Could we focus on v1 in this PR?

Reply:
> Could we focus on v1 in this PR?

Yes

> Unfortunately, v2 hasn't handled it. I am working on it.

Would you mind opening an issue to track the progress?

Reply:
Sure, https://github.com/ray-project/ray/issues/51321